yoticlient.go

```diff
@@ -171,6 +181,12 @@ func getActivityDetails(requester httpRequester, encryptedToken, sdkID string, k
 					}
 				}
 			}
+			formattedAddress, err := getFormattedAddressIfAddressIsMissing(result)
+			if err == nil {
+				result.Address = formattedAddress
```

Comment:
Does this mean if both `structured_postal_address.formatted_address` and `postal_address` are set. It will always return `formatted_address`?

Or this logic should be used directly on line `138`

Reply:
Good spot, I removed that logic in the last commit as I was trying to trim it down, didn't think of that. 

Can't put it on line 138 as there's no guarantee that the structured_postal_address would have been process by then, but I'll change this logic here